compiler/src/iree/compiler/Dialect/Flow/Transforms/Passes.cpp

```diff
@@ -403,11 +358,18 @@ void buildFlowTransformPassPipeline(OpPassManager &passManager,
 }
 
 void registerFlowTransformPassPipeline() {
-  PassPipelineRegistration<TransformOptions> transformPassPipeline(
+  PassPipelineRegistration<> preprocessingTransformPassPipeline(
```

Comment:
Is this just a transient place you want these? We can probably move this also out of Flow. I think everything upto elementwise fusion can be moved out into a global optimization pipeline?
One caveat though is the "type demotion" passes (the i64 -> i32, etc.). Those probably need to happen at the start of flow, and then we go straight into elementwise fusion.

Reply:
I think we still want to keep elementwise fusion cleanups in flow pipeline. I moved everything above global_optimization to the new phase.